warehouse/templates/manage/account/publishing.html

```diff
@@ -94,7 +94,7 @@ <h2>{% trans %}Add a new pending publisher{% endtrans %}</h2>
 
 
   <p class="no-bottom-padding">
-    {% trans href="/help#trusted-publishers" %}
+    {% trans href="https://docs.pypi.org/trusted-publishers/" %}
```

Comment:
Need to remove the preview feature docs? https://github.com/pypi/warehouse/blob/7340873aa5be3da62c4d3cfba273decd090cdd9e/docs/user/main.py#L4-L16

Reply:
Moar docs: https://github.com/pypi/warehouse/blob/7340873aa5be3da62c4d3cfba273decd090cdd9e/docs/user/trusted-publishers/index.md?plain=1#L9-L22 and associated assets

Reply:
Aha, yes, I knew I forgot something.